docs/src/components/PresentationBlock/PresentationBlock.js

```diff
@@ -26,35 +26,31 @@ const getClassName = cssModules(STYLES);
 
 const PresentationBlock = props => {
   const classNames = [getClassName('bpkdocs-presentation-block')];
-  const { className, darkBackground, whiteBackground, ...rest } = props;
```

Comment:
Most components were setting `whiteBackground`, so I decided to make it the default behaviour to reduce the need to pass it in.